pkgs/development/python-modules/colormath/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, fetchPypi
+, networkx
+, numpy
+, lib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "colormath";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3d4605af344527da0e4f9f504fad7ddbebda35322c566a6c72e28edb1ff31217";
+  };
+
+  # Are there no tests?
+  doCheck = false;
```

Comment:
please checkout from github to get tests, looks like they use nose

```
  checkInputs = [ nose ];
  checkPhase = ''
    nosetests
  '';
```

Reply:
Cool thanks, I changed this to use nose tests like that, and tests seem to be passing.